### DISCUSSION TITLE
脚本下载最高清晰度的视频会卡住

### DISCUSSION THREAD

**Discussion by liansekongyk** [the1812/Bilibili-Evolved#2569] [提问 / Q&A]
第一次卡住是在大概一周前，今天下载视频也卡住不动了，A视频最高清晰度是1080P60的，下不了，只能下载1080P的，B视频最高是4K，只能下载1080P
![image](https://user-images.githubusercontent.com/95323015/144150523-69a4ae49-8c7d-4c86-bc1d-d586efd619e7.png)

**Comment by liansekongyk** [comment]
对不起我的问题
我把脚本开的只有这一个就好了，4K也可以点击直链下载
但是我每次使用IDM导出直链，在IDM新增任务之后显示我没有权限下载文件，你们是这样的吗

**Reply by the1812** [reply]
IDM 导出不是新增任务来用的, 是把那个 ef2 文件导进去
![image](https://user-images.githubusercontent.com/26504152/144272128-fc0af67c-77eb-45f1-969f-978a4c12cee7.png)

**Reply by liansekongyk** [reply]
我成功使用这个导入下载文件了
但今天我下载B站视频又出现在我截图的情况
别的相关脚本我一个都没有开
但速度仍然是零
只能导出后用idm下载
请问这个是什么情况

**Reply by the1812** [reply]
不奇怪, `直接下载` 的 bug 一大堆, 已经不再维护了, 以后都用导出吧
